client/src/containers/Homepage.js

```diff
@@ -1,50 +1,68 @@
 import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import { fetchTodaysMeals } from '../actions/index';
+import _ from 'lodash';
+
+import Carousel from 'nuka-carousel';
+import Moment from 'react-moment';
```

Comment:
Please could you get rid of Moment import statement bc we are not using it in this file, and all Moment related code i.e package.json and other places